psock.js

```diff
@@ -11,6 +11,7 @@ const nopt = require('nopt')
 const fs = require('fs')
 
 module.exports = transport
+module.exports.cli = cli
 
 if (require.main === module) {
```

Comment:
This code would be redundant. I think the appropriate course of action would be to remove the `app.js` file and migrate that code here.